include/plugin.php

```diff
@@ -353,12 +359,17 @@ function get_theme_info($theme){
 		'unsupported' => false
 	);
 
-	if (file_exists("view/theme/$theme/experimental"))
+	if (file_exists("view/theme/$theme/experimental")) {
 		$info['experimental'] = true;
-	if (file_exists("view/theme/$theme/unsupported"))
+	}
+
+	if(file_exists("view/theme/$theme/unsupported")) {
```

Comment:
Args, you cannot merge arrays with a `+`. You must use `array_merge()` for it.

Sure I will add it.